presto-main/src/main/java/com/facebook/presto/sql/gen/FunctionCallCodeGenerator.java

```diff
@@ -49,7 +49,7 @@ public BytecodeNode generateCall(FunctionHandle functionHandle, BytecodeGenerato
         }
 
         return context.generateCall(
-                functionManager.getFunctionMetadata(functionHandle).getName().getSuffix(),
+                functionManager.getFunctionMetadata(functionHandle).getName().getUnqualifiedName(),
```

Comment:
I think I mentioned before that `getSuffix` is more accurate than `getUnqualifiedName`. If you prefer to use "unqualified name", it's more appropriate to have a utility function `unqualifyName(qualifiedName)`. I'm not sure whether you can call the last part of a qualified name "unqualified name".

Reply:
It is a convention to called the last part of a "fully qualified object" (file path, class name, etc) a unqualified name. Here are some reference:
1. https://stackoverflow.com/a/27966682
2. https://en.wikipedia.org/wiki/Fully_qualified_name

Reply:
Again, those reference only said a name without prefix is a unqualified name, rather than saying the last part of a qualified name is a "unqualified name". The reference you mentioned also used "suffix", which I still think is better than "unqualified name". If you want to be stubborn on this I won't insist. But I think "suffix" is better than "unqualified name" in this case.

Reply:
Looking at some of our existing classes:
- `QualifiedObjectName` is defined as (catalogName, schemaName, objectName)
- `QualifiedTablePrefix` is defined as (catalogName, schemaName, tableName)

What about defining `QualifiedFunctionName` is (functionNamespace, functionName)?